modules/util/portal-tools-bundle-support/src/main/java/com/liferay/portal/tools/bundle/support/maven/CreateTokenMojo.java

```diff
@@ -70,7 +70,7 @@ public void execute() throws MojoExecutionException {
 	)
 	protected File tokenFile;
 
-	@Parameter(defaultValue = BundleSupportConstants.DEFAULT_BUNDLE_URL)
+	@Parameter(defaultValue = BundleSupportConstants.DEFAULT_TOKEN_URL)
```

Comment:
@david-truong, does this look like the right fix to you?

Reply:
Ah yes, the wrong constant.